rafx-renderer/src/swapchain_render_resource.rs

```diff
@@ -37,11 +44,21 @@ impl SwapchainResources {
             )
             .ok_or_else(|| "Could not find a supported depth format")?;
 
-        Ok(SwapchainResources {
+        self.0 = Some(SwapchainRenderResourceInner {
             swapchain_surface_info,
             default_color_format_hdr,
             default_color_format_sdr,
             default_depth_format,
-        })
+        });
+
+        Ok(())
+    }
+
+    pub fn clear_swapchain(&mut self) {
+        self.0 = None;
+    }
+
+    pub fn get(&self) -> Option<&SwapchainRenderResourceInner> {
```

Comment:
Can we rename this to `try_get` and add a separate `get` that performs the unwrap, then change the current uses of `try_get` to the `get` call instead?

Reply:
yeah